cmd/controllers/app/server.go

```diff
@@ -72,10 +76,17 @@ func Run(opt *options.ServerOption) error {
 		return err
 	}
 
-	jobController := job.NewJobController(config)
+	// TODO: add user agent for different controllers
```

Comment:
What is user agent?

Reply:
It is just for http server to know the client identity.

Reply:
A http header